### PR TITLE
[WIP] Use Sistent error boundaries

### PR DESCRIPTION
**Pull request by Yashsharma1911**
**Notes for Reviewers**

This PR fixes #

This PR replaces error boundaries defined in MeshMap with error boundaries in Sistent to be consistent

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by Yashsharma1911** [comment]
This PR needs Sistent new release after that it's good to go

File: ui/components/Modals/SupportModal.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+import Modal from '../Modal';
+import PublicIcon from '@material-ui/icons/Public';
+import { useNotification } from '@/utils/hooks/useNotification';
+import { connect } from 'react-redux';
+import { EVENT_TYPES } from '../../lib/event-types';
+import axios from 'axios';
+
+const helpModalSchema = {
+  title: 'Support Form',
+  properties: {
```
[the remark this answers is not in the document]
Comment (leecalcote) [reply ui/components/Modals/SupportModal.js:11]: Ok.

**Comment by leecalcote** [comment]
> This PR needs Sistent new release after that it's good to go

@nebula-aac a new Sistent release is needed. I assume that there's no special instructions other than the commonplace need to update the version number and click "release", right?

**Comment by Yashsharma1911** [comment]
> > This PR needs Sistent new release after that it's good to go
> 
> @nebula-aac a new Sistent release is needed. I assume that there's no special instructions other than the commonplace need to update the version number and click "release", right?

Thankyou for the release, I'll confirm and merge this today

**Comment by leecalcote** [comment]
Ready to rock 'n roll...

**Comment by Yashsharma1911** [comment]
> Ready to rock 'n roll...

Yes we are more likely good to go here, however, I just need to replace existing error boundary with sistent, on it

**Comment by leecalcote** [comment]
@nebula-aac does this look right?

**Pull request closed without merging**